resources/js/layouts/LandingLayout.vue

```diff
@@ -165,3 +168,20 @@ export default {
     }
 }
 </script>
+
+<style scoped>
+/*skip to main content*/
+.skip-nav-link {
+    background: yellow;
+    padding: 0.5rem 1.5rem;
+    border-radius: 0 0 0.25rem 0.25rem;
+    position: absolute;
+    left: 0.5rem;
+    transform: translateY(-200%);
+    transition: transform 325ms ease-in;
+}
+
+.skip-nav-link:focus {
+    transform: translateY(-60%);
+}
+</style>
```

Comment:
instead of having the style block the same in two files use a sass file for the styling

Reply:
Unfortunately, this is not possible because the parameters for each layout are individual.